Meta/build-image-extlinux.sh

```diff
@@ -8,7 +8,7 @@ script_path=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
 
 if [ "$(id -u)" != 0 ]; then
     set +e
-    ${SUDO} -- sh -c "\"$0\" $* || exit 42"
+    ${SUDO} ${SHELL} -c -- "\"$0\" $@ || exit 42"
```

Comment:
shellcheck is not happy about this and asks for some quotes, but I don't know if that's correct or would break :shrug:

Reply:
I don't see a reason why it would break. But, shellcheck is also not happy about the `$*` to `$@` change.

Reply:
I don't even remember the full difference between those two, my muscle memory just said that it should be a $@. I can change it back, prolly.

Reply:
fwiw `$@` gives you a list, and `$*` gives you the stringified version of that list
unquoted they're mostly the same, they also behave the same way in this string composition context.